### DISCUSSION TITLE
Leanback

### DISCUSSION THREAD

**Discussion by StaehliJ** [SRGSSR/pillarbox-android#404] [Polls]
The default integration of Leanback provided by Media3 is very simple (only play/pause actions).  Pillarbox could add more actions and playback feature support like playback speed, and tracks selections.

We don't think it is a good idea as Leanback is no more maintained by Google.

**Comment by MGaetan89** [comment]
Leanback has been removed in https://github.com/SRGSSR/pillarbox-android/pull/651, and Compose TV is now the recommended way.
